app/javascript/containers/PortfolioBuildingContainer.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PortfolioBuildingInfoContainer from './PortfolioBuildingInfoContainer';
+import CategoryDetailsContainer from './CategoryDetailsContainer';
+import { get } from '../fetch/requester';
+import { connect } from "react-redux";
+import { questionDataPerCategory } from '../selectors/answersSelector';
+
+class PortfolioBuildingContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {userData: null};
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.selectedBuildingId !== this.props.selectedBuildingId) {
+          this.getUserData();
+        }
+    }
+
+    async getUserData() {
```

Comment:
Let's rename this to `getUserLoginData()` so it's more clear what information we're actually looking for (I can see this possibly getting confused with retrieving data on the currently logged-in user)